named_entity_recognition/documentation/step-3-upload-dataset.md

```diff
@@ -0,0 +1,30 @@
+# Dataset
+
+## Upload dataset
+Please [download](https://drive.google.com/open?id=1qNNaYguyH_xLfbqnR5ABuYrjblkUEr_z) the dataset and upload it to a Google Cloud Storage location. 
```

Comment:
@texasmichelle It would be great if someone of you can support me and upload the dataset to the kubeflow specific bucket. (since I don’t have any access)

Just send me the link I will add it to the documentation and adapt the command accordingly.

Reply:
✓ done (added a direct link to the kubeflow example bucket and removed upload step since we can use the file directly from the bucket)

Reply:
Done! Uploaded to that location